FLAnimatedImage/FLAnimatedImageView.m

```diff
@@ -9,6 +9,7 @@
 
 #import "FLAnimatedImageView.h"
 #import "FLAnimatedImage.h"
+#import "FLWeakProxy.h"
```

Comment:
Is it required?

Reply:
Yes. It seems FLAnimateImageView use it to avoid retain cycle when using CADisplayLink.
There's a comment about it.
```
// It is important to note the use of a weak proxy here to avoid a retain cycle. `-displayLinkWithTarget:selector:`
// will retain its target until it is invalidated. We use a weak proxy so that the image view will get deallocated
// independent of the display link's lifetime. Upon image view deallocation, we invalidate the display
// link which will lead to the deallocation of both the display link and the weak proxy.
```